packages/react-json-tree/src/index.tsx

```diff
@@ -7,12 +7,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import JSONNode from './JSONNode';
 import createStylingFromTheme from './createStylingFromTheme';
-import {
-  invertTheme,
-  type StylingConfig,
-  type StylingFunction,
-  type StylingValue,
-  type Theme,
+import invertTheme from 'react-base16-styling';
```

Comment:
```suggestion
import { invertTheme } from 'react-base16-styling';
```

Reply:
Hey @Methuselah96 made the changes. Review it now.